src/components/SendFundsForm.vue

```diff
@@ -283,7 +284,18 @@ export default {
   },
   emits: ['send', 'error'],
   setup() {
+    const store = useStore()
+
+    const isOnline = computed(() => {
+      return store.getters['isOnline']
+    })
+
+    const checkIsOnline =  () => {
+      return navigator.onLine || isOnline.value;
+    }
```

Comment:
Prettify: extra space

```suggestion
    const checkIsOnline = () => {
      return navigator.onLine || isOnline.value;
    }
```